test/support/subclasses.rb

```diff
@@ -0,0 +1,6 @@
+# Extract the subclasses that exist early on because other classes will get added by tests
+#   later on in execution and in an unpredictable order.
+class ActiveSupport::TestCase
+  cattr_accessor :measurable_subclasses
+  self.measurable_subclasses = Measured::Measurable.subclasses
```

Comment:
This extracts the existing subclasses to use later. Otherwise we get the `fake_system` stuff included here which we don't want to test.